AggregateMetrics/AggregateMetrics.Tests/AzureWebApp/AzureWebAppTest.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace webAppTest
+{
+    using System;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+    using Microsoft.ApplicationInsights.Extensibility.AggregateMetrics.AzureWebApp;
+    using Microsoft.ApplicationInsights.DataContracts;
+    using System.Diagnostics;
+    [TestClass]
+    public class UnitTestAzureWeb
+    {
+        [TestMethod]
```

Comment:
I would encourage you to strive for StyleCop compliant code when adding net new files, in particular writing the doc comments for future maintainers to read.